docs/api/profiles.md

```diff
@@ -0,0 +1,86 @@
+# Profiles
+
+Profiles contain information about a User. A User does not need a Profile, but a Profile is always created
+together with a User.
```

Comment:
Why create a profile with a user? Or do you mean that a profile always needs a user?